File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main (void){
     int n;
    do{
        printf("Height:");
        n = get_int();
    }
    while((n < 0) || (n > 23));

 int row;
    for( row = 1; row < n + 1; row ++ ){
       int space;
       int hash;

       space = n - row;
       hash = n - space;

       int i;
       for (i = space; i > 0; i--){
           printf(" ");
       }
       for (i = hash; i > 0; i--){
           printf("#");
       }
       printf("  ");
       for (i = row; i > 0; i--){
           printf("#");
       }
       printf("\n");
   }
}

File: pset6/mario.py
import cs50

while True:
    print('Height:', end="")
    height = cs50.get_int()
    if height >= 0 and height <= 23:
        break

stringlen = int(height) + 1
hashes = 2
spaces = stringlen - hashes

for i in range(height):

    print(spaces * ' ', end="")
    print(hashes * '#', end="")
    print()

    hashes += 1
    spaces -= 1
